draft-ietf-httpbis-incremental.md

```diff
@@ -142,6 +142,26 @@ generally more consistent with HTTP's architecture.
 
 # Security Considerations
 
+When receiving an incremental request, intermediaries might reject the request
+due to security concerns. The following subsections explore typical scenarios
+under which the intermediaries might reject the requests.
+
+
+## Permanent Rejection
+
+Some intermediaries inspect the payload of an HTTP request and forward them only if
+the request is deemed safe. Such intermediaries might decide to reject
+incremental requests that do not carry sufficient information in its first
+flights of payload necessary to determine the safety of the request.
+
+When an intermediary rejects an incremental request due to security concerns
+with regard to the payload that the request might convey, the intermediary
+SHOULD respond with a 403 Forbidden error with an http_request_denied Proxy-Status
+response header field ({{Section 2.3.17 of PROXY-STATUS}}).
```

Comment:
There are two cases here:

1. incremental requests, which this text assumes and for which 403 etc is fine.
2. incremental *responses*, which I think also get some sort of rejection, but it won't be a http_request_denied, maybe http_response_timeout with 504.

The suggestions above try to make that text more generic, but I don't quite know how to make this text more general without spelling that out.

Reply:
+1 to making the problem statement generic, so I've applied your suggestions against the previous paragraph.

Regarding what the signal should be, I think I'm more inclined to defining the `incremental_unsupported` Proxy-Status error, as that can be the source of truth for the rejection of both requests and responses.

Reply:
Added as `incremental_refused` in commits up to 4b4bfc7.